cli/src/CliProcessor.cpp

```diff
@@ -272,8 +272,11 @@ void CliProcessor::setupArgParsing(int argc, const char* const* argv)
 void CliProcessor::startOSNProcessing()
 {
     OpenSpaceNet osn(std::move(osnArgs));
-    pd_ = boost::make_shared<ConsoleProgressDisplay>();
-    osn.setProgressDisplay(pd_);
+
+    auto pd = boost::make_shared<ConsoleProgressDisplay>();
+    osn.setProgressDisplay(pd);
```

Comment:
I believe you can `move` `pd` here (given that you move the def of enable timing downstream).

Reply:
I cannot. I need `osn.setProgressDisplay` to initialize categories before I can enable timing, so I have to hold on to the pointer.